tuxemon/event/actions/transfer_money.py

```diff
@@ -15,50 +15,55 @@
 @dataclass
 class MoneyMathAction(EventAction):
     """
-    Performs a mathematical transaction on the player's money.
+    Performs a money transaction between entities.
 
     Script usage:
         .. code-block::
 
             transfer_money <transaction>,<amount>[,slug]
 
     Script parameters:
+        slug1: Slug name (e.g. NPC, etc.)
         transaction: Operator symbol.
         amount: amount of money.
-        slug: Slug name (e.g. NPC, etc.)
-        eg: +,100,mom (player gets 100 from mom)
-        eg: -,100,mom (mom gets 100 from player)
-        eg: +,100 (player gets 100)
+        slug2: Slug name (e.g. NPC, etc.)
+
+    eg: player,+,100,mom (player gets 100 from mom)
+    eg: player,-,100,mom (mom gets 100 from player)
+    eg: player,+,100 (player gets 100)
 
     """
 
     name = "transfer_money"
+    slug1: str
     transaction: str
     amount: int
-    slug: Union[str, None] = None
+    slug2: Optional[str] = None
 
     def start(self) -> None:
         player = self.session.player
 
         # Read the parameters
         transaction = self.transaction
         amount = self.amount
-        wallet_player = player.money.get("player", 0)
+        wallet_player = player.money.get(self.slug1, 0)
```

Comment:
would we rename the `wallet_` variables here?  i think its great that we are allowing the walllets to change, but if the variable is still named "player", the code becomes difficult to understand

Reply:
I agree with you

Reply:
Ok, wallet_character1 and wallet_character2